lib/line-numbers.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const callsites = require('callsites');
+const micromatch = require('micromatch');
+const sourceMapSupport = require('source-map-support');
+const flatten = require('lodash/flatten');
+const parseTestSourceInFile = require('./test-ast');
+
+const NUMBER_REGEX = /^-?\d+$/;
+const RANGE_REGEX = /^(?<startGroup>-?\d+)-(?<endGroup>-?\d+)$/;
+const LINE_NUMBERS_REGEX = /^(?:-?\d+(?:--?\d+)?,?)+$/;
+const DELIMITER = ':';
+
+const pluralizeIf = (string, condition) => condition ? `${string}s` : string;
```

Comment:
We already use https://www.npmjs.com/package/plur for this.

Reply:
Ah, cool! Good to know. Although, judging by the comment below neither should be needed any longer?

Reply:
Just figured I'd share it regardless 😄